src/scripts/restore.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/env bash
+
+restore() {
+	if [ -z "${SKPR_BACKUP}" ]; then
+		SKPR_BACKUP=$(skpr backup list "${SKPR_SOURCE}" --json | jq -r '[.][0][0].Name')
+		if $? -gt 0; then
+			echo "Failed to get name of most recent backup"
+			exit 1
+		fi
+		# shellcheck disable=SC2086
+		echo "export SKPR_BACKUP=${SKPR_BACKUP}" >>$BASH_ENV
+	fi
+	echo "Restoring backup '${SKPR_BACKUP}' to ${SKPR_TARGET}"
+
+	if [ -z "${SKPR_SOURCE}" ] || [ -z "${SKPR_TARGET}" ]; then
```

Comment:
Split this out to check one at a time?

Reply:
The error bundles both together and might get confusing for an end user.

Reply:
Separated out now.